docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -9,46 +9,48 @@ for the executable on the command line. The one exception is [the
 bootstrap script](#bootstrap), which is designed to be fetched from
 github and fed directly into `bash`.
 
-There are two ways of deploying kcp and the central KubeStellar
-components: (1) as processes on a machine of supported OS and ISA, and
+This document refers to KCP as a space provider that KubeStellar can use. 
+There are two ways of deploying the space provider in this 
```

Comment:
The injection of "in this document," in this sentence is unnecessary and (I suspect) not really intended and messes up the logical structure of the sentence (look at the dangling clause about central KubeStellar components).

Reply:
wouldn't it be clearer to preface with something like:

> **Note:** KubeStellar can use any of several space providers "under the covers." At the time of this PoC (early 2023) our implementation focused on kcp as that space provider.

And then continue with the "There are two ways... "